lms/templates/edxnotes/note-item.underscore

```diff
@@ -37,7 +37,9 @@
 
         <% if (tags.length > 0) { %>
             <h3 class="reference-title"><%- gettext("Tags:") %></h3>
-            <span class="reference-meta reference-tags"><%- tags.join(", ") %></span>
+            <% for (var i = 0; i < tags.length; i++) { %>
+                <a class="reference-meta reference-tags" href="#"><%- tags[i] %></a>
```

Comment:
@marcotuts What do you think about @cptvitamin's suggestion to get rid of role="complementary" aria-label="<%- gettext('References') %>" on footer and converting h3 elements to p elements?